package.json

```diff
@@ -21,18 +21,22 @@
     ]
   },
   "dependencies": {
+    "@types/imagemin-svgo": "7.0.0",
+    "@types/svg-sprite-loader": "3.9.2",
```

Comment:
Should add these to dev deps instead, same in the package-lock.json

Reply:
These dependencies weren't needed (I added them to try to fix the typing of next-optimized-images but forgot to remove them)